python/ray/util/queue.py

```diff
@@ -12,14 +12,32 @@ class Full(Exception):
 
 
 class Queue:
-    """Queue implementation on Ray.
+    """A first-in, first-out queue implementation on Ray.  The behavior and use
+    cases are similar to those of the asyncio.Queue class.
```

Comment:
Convention in a docstring is for the first line to be a one-line description (not wrap)

```suggestion
    """A first-in, first-out queue implementation on Ray. 
    
    The behavior and use cases are similar to those of the asyncio.Queue class.
```